lib/uksched/include/uk/thread.h

```diff
@@ -141,18 +141,29 @@ struct uk_thread *uk_thread_current(void)
 #define UK_THREADF_UKTLS      (0x002)	/**< Unikraft allocated TLS */
 #define UK_THREADF_RUNNABLE   (0x004)
 #define UK_THREADF_EXITED     (0x008)
+/*
+ *  A flag used for marking that a thread could potentially
+ *  be added to the run queue. A thread marked as such
+ *  must not be running and already present in run queue.
```

Comment:
When I first read the comment I understood it as: `A thread marked as such (must not be running) and (already present in run queue).`
Might be less ambiguous using "neither .. nor ...".